libraries/AP_HAL_ChibiOS/RCOutput.cpp

```diff
@@ -728,6 +728,53 @@ void RCOutput::set_output_mode(uint16_t mask, enum output_mode mode)
 #endif
 }
 
+/*
+ * get output mode banner to inform user of how outputs are configured
+ */
+bool RCOutput::get_output_mode_banner(char banner_msg[], uint8_t banner_msg_len) const
+{
+    // create array of each channel's mode
+    output_mode ch_mode[NUM_GROUPS * ARRAY_SIZE(pwm_group::chan)] = {MODE_PWM_NONE};
```

Comment:
This is only initialising the first element as `MODE_PWM_NONE` and the rest as zeroes.  Happens to be the same value, but we shouldn't rely on it.

Reply:
I did not realise this!  I've changed it to just "= {}" which should initialise it to zero I think which is what we want.  txs!